src/app/config/statefile.c

```diff
@@ -71,8 +71,10 @@ static config_abbrev_t state_abbrevs_[] = {
 DUMMY_TYPECHECK_INSTANCE(or_state_t);
 
 /*XXXX these next two are duplicates or near-duplicates from config.c */
```

Comment:
Will these duplicates will be refactored in some future PR?

Same question for shared random and other modules.

Reply:
Yes: also part of #30935.